### ISSUE TITLE
RFE: Vertical Fractions

### ISSUE THREAD

**Issue by TPS** [lights0123/fractions#5]
@lights0123 1st of all, very cool tool!  I think it'd be amazing if it could be extended to optionally make vertical fractions, à la [Wikipedia's `sfrac` template](https://en.wikipedia.org/wiki/Template:Sfrac).  Is it possible with just Unicode?

**Comment by mrienstra** [comment]
My impression is that it's not currently possible with Unicode, but I could be wrong.

**Comment by quincynyan** [comment]
Yes that is correct, because this isn't Unicode, but rather rendered by Wikipedia's rendering tool. If you inspect (ctrl + shift + i), you can clearly see they're just a bunch of spans and divs. 
![image](https://user-images.githubusercontent.com/65074780/223638012-9d7e4e4f-9422-46e4-9687-6281c8145cc2.png)

**Comment by TPS** [comment]
Given how outdated much of Wikipedia's software is vs Unicode updates/improvements, I'm still hoping this can be done directly in Unicode.

**Comment by quincynyan** [comment]
Well, there isn't really such a thing yet:
https://www.google.com/search?q=unicode+fraction
https://en.wikipedia.org/wiki/Number_Forms
